src/Resources/views/includes/layout/_block.html.twig

```diff
@@ -24,7 +24,9 @@
             {% endif %}
             <div class="{% if block.option('label') or block.option('description') %}border-t border-neutral-200 bg-white sm:rounded-b-lg{% endif%}">
                 <dl>
-                    {{ content }}
+                    {% for content in block.contents(page)|filter(c => is_granted(c.option('show_voter_attribute'), view.data)) %}
```

Comment:
extract filter into twig function. depending PAGE different voter attribute. attribute can be null or false, needs to be ignored in this case.

Reply:
```
Variable "page" does not exist.
```